modules/test/playwright/fixtures/objectPagesTest.ts

```diff
@@ -8,18 +8,38 @@ import {test} from '@playwright/test';
 import {ModalAddObjectDefinitionPage} from '../pages/object-web/ModalAddObjectDefinitionPage';
 import {ModelBuilderPage} from '../pages/object-web/ModelBuilderPage';
 import {ViewObjectDefinitionsPage} from '../pages/object-web/ViewObjectDefinitionsPage';
+import { EditObjectDefinitionPage } from '../pages/object-web/EditObjectDefinitionPage';
+import { SidePanelObjectActionPage } from '../pages/object-web/object-action/SidePanelObjectActionPage';
+import { ViewObjectActionsPage } from '../pages/object-web/object-action/ViewObjectActionsPage';
+import { ActionBuilderPage } from '../pages/object-web/object-action/ActionBuilderPage';
 
 const objectPagesTest = test.extend<{
```

Comment:
Hey, we are mixing too much pages here, I guess. Model Builder pages and Edit Object Definition pages should have their own fixtures. Can you create a new one, like `editObjectDefinitionPagesTest.ts`?